src/main/java/net/snowflake/ingest/streaming/internal/IcebergParquetValueParser.java

```diff
@@ -161,12 +173,16 @@ private static ParquetBufferValue parseColumnValueToParquet(
             subColumnFinder,
             defaultTimezone,
             insertRowsCurrIndex,
-            isDescendantsOfRepeatingGroup);
+            isDescendantsOfRepeatingGroup,
+            error);
       }
     }
 
     if (value == null) {
       if (type.isRepetition(Repetition.REQUIRED)) {
+        if (error != null) {
```

Comment:
call getDotPath once and reuse it in both line 184 and 191.

Reply:
Called once at the start.